source/features/one-key-formatting.tsx

```diff
@@ -7,24 +7,32 @@ import {onCommentFieldKeydown, onConversationTitleFieldKeydown, onCommitTitleFie
 
 const formattingCharacters = ['`', '\'', '"', '[', '(', '{', '*', '_', '~', '“', '‘'];
 const matchingCharacters = ['`', '\'', '"', ']', ')', '}', '*', '_', '~', '”', '’'];
+const quoteCharacters = new Set(['`', '\'', '"']);
 
 function eventHandler(event: DelegateEvent<KeyboardEvent, HTMLTextAreaElement | HTMLInputElement>): void {
 	const field = event.delegateTarget;
+	const formattingChar = event.key;
 
-	if (!formattingCharacters.includes(event.key)) {
+	if (!formattingCharacters.includes(formattingChar)) {
 		return;
 	}
 
 	const [start, end] = [field.selectionStart, field.selectionEnd];
 
 	// If `start` and `end` of selection are the same, then no text is selected
-	if (start === end) {
+	if (start === end || start === null || end === null) {
```

Comment:
Did you encounter `null` when testing or is it just a type issue? You can use `start!` to silence it.

Reply:
It was only typing. But always silencing it seemed wrong, I mean the type includes `null` for a reason I guess?

Reply:
DOM types consider more than the usual. Even .textContent is string | null, but that only applies to image tags, so silencing them is correct in many cases. 

Also in extensions things change under your feet so we prefer throwing errors when things don't match our expectations instead of just silencing everything like you would in a self-contained application.